packages/vega-typings/types/spec/transform.d.ts

```diff
@@ -252,6 +253,17 @@ export interface JoinAggregateTransform {
   as?: (string | SignalRef | null)[] | SignalRef;
 }
 
+export interface KDETransform {
+  type: 'kde';
+  field: string | TransformField | SignalRef;
+  groupby?: (string | TransformField)[] | SignalRef;
+  method?: 'pdf' | 'cdf' | SignalRef;
+  bandwidth?: number | SignalRef;
+  extent?: number[] | SignalRef;
```

Comment:
```suggestion
  extent?: [number, number] | SignalRef;
```

Reply:
Thanks! I've pushed some minor refinements to the transform typings in response.